dir.props

```diff
@@ -36,8 +36,7 @@
   <PropertyGroup>
     <BaseOutputPath>$(ProjectDir)bin/</BaseOutputPath>
     <ToolsDir>$(ProjectDir)Tools/</ToolsDir>
-    <TaskDirectory>$(ProjectDir)tools-local/tasks/</TaskDirectory>
-    <TasksBinDir>$(TaskDirectory)Microsoft.DotNet.SourceBuild.Tasks/bin/Debug/netstandard1.5/</TasksBinDir>
+    <TasksBinDir>$(ProjectDir)bin/AnyCPU/Debug/netstandard1.5/</TasksBinDir>
```

Comment:
Isn't that going to be the wrong output for all other projects?

Reply:
No.  This should be correct for all projects.

Reply:
We are now putting the task dll directly in that directory? It seems a little odd that we no longer put it under a directory with it's name in the path.

Reply:
I'm going to update OutputPath to put it back under the tasks directory.